.github/workflows/bump-dependencies.yml

```
@@ -0,0 +1,36 @@
+name: Bump dependencies
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '0 8 * * 1' # At 08:00 on Monday.
+
+jobs:
+  bump-dependencies:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+
+      - run: python -m pip install poetry pre-commit
+      - run: poetry lock
+      - run: poetry export --without-hashes --dev -f requirements.txt -o requirements_dev.txt -E python
+      - run: poetry export -f requirements.txt --without-hashes -o requirements.txt
+      - run: pre-commit autoupdate
+      - run: git status
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
```

Use the hash to freeze the tag as it is a 3rd party dependency

```suggestion
        uses: peter-evans/create-pull-request@18f7dc018cc2cd597073088f7c7591b9d1c02672
```